dashboard/app/controllers/maker_controller.rb

```diff
@@ -113,6 +113,16 @@ def schoolchoice
   def login_code
   end
 
+  # GET /maker/display_code
+  # renders a page for users to copy and paste a login key
+  def display_code
+    # Generate encrypted code to display to user
+    auth_user = current_user.authentication_options.find_by_credential_type(AuthenticationOption::GOOGLE)
```

Comment:
super nit: this is an `AuthenticationOption`, not a `User`, so we could rename this to something like `user_auth` or just `auth_option`